checker_test.go

```diff
@@ -0,0 +1,384 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	qt "github.com/frankban/quicktest"
+)
+
+var numArgsTests = []struct {
+	checker         qt.Checker
+	expectedNumArgs int
+}{{
+	checker:         qt.Equals,
+	expectedNumArgs: 1,
+}, {
+	checker:         qt.CmpEquals(),
+	expectedNumArgs: 1,
+}, {
+	checker:         qt.DeepEquals,
+	expectedNumArgs: 1,
+}, {
+	checker:         qt.ErrorMatches,
+	expectedNumArgs: 1,
+}, {
+	checker:         qt.PanicMatches,
+	expectedNumArgs: 1,
+}, {
+	checker: qt.IsNil,
+}, {
+	checker:         qt.Not(qt.Equals),
+	expectedNumArgs: 1,
+}}
+
+func TestNumArgs(t *testing.T) {
```

Comment:
tbh I'm not sure that I'd bother with this test - it would be possible to add more checker tests that cover the expected argument count, I think, and then the tests would still be useful even if the Checker interface changed.

Reply:
Done.